Sources/NIOHTTP2/HTTP2ChannelHandler.swift

```diff
@@ -103,7 +103,7 @@ public final class NIOHTTP2Handler: ChannelDuplexHandler {
 
     /// The delegate for (de)multiplexing inbound streams.
     private var inboundStreamMultiplexerState: InboundStreamMultiplexerState
-    internal var inboundStreamMultiplexer: InboundStreamMultiplexer? {
+    internal var inboundStreamMultiplexer: NIOLoopBound<InboundStreamMultiplexer>? {
```

Comment:
Why does this need to be a `NIOLoopBound`?

Reply:
It doesn't, removed.